test/e2e/autoscale.go

```diff
@@ -308,11 +308,11 @@ func checkPodScale(ctx *testContext, targetPods, minPods, maxPods float64, durat
 				return errors.New("interim scale didn't fulfill constraints: " + mes)
 			}
 			// A quick test succeeds when the number of pods scales up to `targetPods`
-			// (and, for sanity check, no more than `maxPods`).
+			// (and no more than `maxPods`).
 			if got >= targetPods && got <= maxPods {
 				if quick {
 					// A quick test succeeds when the number of pods scales up to `targetPods`
-					// (and, for sanity check, no more than `maxPods`).
+					// (and no more than `maxPods`).
```

Comment:
is this whole block some sort of copy paste error? seems like we re-do the outer if - and dupe the comment - for no reason (I know you didn't introduce this, just wondering!)

Reply:
Indeed, simplified.